### PR TITLE
feat: content field auto convert empty string to None

### PR DESCRIPTION
**Pull request by omg-xtao**

### CODE REVIEW

**Comment by yupix** [comment]
contentにはデフォルトでNoneが入っているため、この変更は意味をなさないように感じます。

**Comment by omg-xtao** [comment]
> Since content contains None by default, this change seems to make no sense.
> 
> https://github.com/yupix/MiPAC/blob/91ab86f439450fbcc48c1dc282415d2465dea5ed/mipac/actions/note.py#L22

It can set the content to None when you pass "".

**Comment by yupix** [comment]
That's true, but I don't know about the recent v12~v13, but I'm a little unsure if that needs to be addressed, since at least in v11 you couldn't use "" for content.

**Comment by yupix** [comment]
I'd like to merge them, but I think this is more like a feat than a fix. To be honest, I feel that this is a problem that can occur when people who are new to Python use keyword arguments without understanding them, and I recognize that this is a countermeasure to that problem. So, as soon as I move CHANGELOG from Fixed to Added, I will merge it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
     - `IAnnouncement` -> `IMetaAnnouncement`
 - `cache` decorator no longer uses `dynamic_args` decorator
 
+### Fixed
+
+- The content field no longer supports empty string.
```
Comment (yupix) [inline CHANGELOG.md:61]: Move from Fixed to Added

**Review by yupix — APPROVED** [review]
💯

**Pull request merged**